src/components/bs5/breadcrumbs/breadcrumbs.hbs

```diff
@@ -1,13 +1,16 @@
-<div class="row">
-    <div class="col-12">
-        <nav aria-label="breadcrumb">
-            <ol class="breadcrumb">
-                {{#each breadcrumbs}}
-                <li class="breadcrumb-item {{linkclass}}">
-                    <a href="{{link}}">{{linktext}}</a>
-                </li>
-                {{/each}}
-            </ol>
-        </nav>
-    </div>
-</div>
\ No newline at end of file
+<nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+        {{#each breadcrumbs}}
+            {{! print all breadcrumbs except the current page as links}}
+            {{#unless @last}}
```

Comment:
If the breadcrumbs object is in pagination/page order, the last array item might not be the "current" item.

Can we use something like #if iscurrent instead? ie Add an iscurrent: true properties of the breadcrumb object?